packages/neuron-wallet/src/block-sync-renderer/sync/indexer-cache-service.ts

```diff
@@ -92,12 +92,12 @@ export default class IndexerCacheService {
           this.indexer,
           {
             lock: {
-              code_hash: lockScript.codeHash,
-              hash_type: lockScript.hashType,
+              codeHash: lockScript.codeHash,
```

Comment:
Is it possible to use `lock: lockScript` here?

Reply:
refactored